src/modules/pages/page_repository.ts

```diff
@@ -49,15 +49,10 @@ export namespace Page {
     updated_at: new Date(),
   })
 
-  export const createFile = async (requestBody: Entity.StructFile) => {
-    const item: any = await Files().where('source', requestBody.source).first()
-    if (item) return Files().where('source', requestBody.source).update(requestBody)
-
-    return Files().insert({
-      ...requestBody,
-      created_at: new Date(),
-    })
-  }
+  export const createFile = async (requestBody: Entity.StructFile) => Files().insert({
+    ...requestBody,
+    created_at: new Date(),
+  })
```

Comment:
_confirm_

so, create file is always insert?

Reply:
yes always create file each insert, because it is already handled on validation unique